zospy/analyses/mtf.py

```diff
@@ -237,3 +237,125 @@ def fft_through_focus_mtf_fromcfg(
     )
 
     return analysis.complete(oncomplete, result)
+
+
+def huygens_mtf(
+    oss: OpticStudioSystem,
+    pupilsampling: str | int = "32x32",
+    imagesampling: str | int = "32x32",
+    imagedelta: float = 0.0,
+    wavelength: str | int = "All",
+    field: str | int = "All",
+    mtftype: constants.Analysis.Settings.Mtf.HuygensMtfTypes | str = "Modulation",
+    maximumfrequency: float = 150.0,
+    use_polarization: bool = False,
+    use_dashes: bool = False,
+    oncomplete: OnComplete | str = OnComplete.Close,
+) -> AnalysisResult:
+    """Wrapper around the OpticStudio Huygens MTF.
+
+    For an in depth explanation of the parameters, see the Zemax OpticStudio user manual
+
+    Parameters
+    ----------
+    oss: zospy.core.OpticStudioSystem
```

Comment:
NumpyDoc requires a space before the colon:

```
    oss : zospy.core.OpticStudioSystem
```

Unfortunately, I have not yet found a way to check this automatically (our formatting tools can't do it, as far as I know), so we still need to change this manually.

Reply:
Is this extra space required in the entire docstring? Existing docstrings in the same file do not have this.

Reply:
I have modified the docstring of `huygens_mtf` accordingly.